src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -82,4 +82,11 @@ public interface CommandExecutor {
         CommandResult execute(String commandText) throws CommandException, ParseException;
     }
 
+    /**
+     * Getter method for command text field
+     */
```

Comment:
I believe adding JavaDoc comments is a diligent thing to do, but I was wondering if they would be necessary for getter methods (since some may argue that getter methods' functionalities are apparent to both the user and developer). I would suggest removing this, but would CheckStyle throw a warning if this comment is removed?

Reply:
JavaDoc is mostly unnecessary for obvious methods like getters and setters. Very unlikely CheckStyle will throw a warning but better safe than sorry.